.travis.yml

```diff
@@ -13,6 +13,8 @@ cache: bundler
   script: false
 
 env:
+  global:
+    - BEARS_ZIP_URL=https://codeload.github.com/coala/coala-bears/zip
```

Comment:
if you look at the Travis log, you will see the moban job isnt running.
It is because of this line.
set the variable in the script above `curl...`